tests/SixLabors.Fonts.Tests/TextLayoutTests.cs

```diff
@@ -668,6 +668,14 @@ public void BuildTextRuns_PreventsOverlappingRun()
     [InlineData(TextDirection.RightToLeft)]
     public void TextJustification_InterCharacter_Horizontal(TextDirection direction)
     {
+        //const string textI = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc ornare maximus vehicula. Duis nisi velit, dictum id mauris vitae, lobortis pretium quam. Quisque sed nisi pulvinar, consequat justo id, feugiat leo. Cras eu elementum dui.";
```

Comment:
I assume left over debugging code...should be removed

Reply:
Yeah... I'll remove it.